cdk/src/monitoring/synthetics.handler.ts

```diff
@@ -0,0 +1,50 @@
+const synthetics = require('Synthetics');
+const log = require('SyntheticsLogger');
+const syntheticsConfiguration = synthetics.getConfiguration();
+
+exports.handler = async function () {
+  // TODO: cycle through several addresses
+  const addresses = ['5607 HOME LN, TOLEDO OH 43623'];
+  const address = addresses[0];
+
+  const url = `https://${process.env.DOMAIN}/`;
+
+  syntheticsConfiguration.setConfig({
+    includeRequestHeaders: true, // Enable if headers should be displayed in HAR
+    includeResponseHeaders: true, // Enable if headers should be displayed in HAR
+    restrictedHeaders: [], // Value of these headers will be redacted from logs and reports
+    restrictedUrlParameters: [], // Values of these url parameters will be redacted from logs and reports
+  });
+  const page = await synthetics.getPage();
+
+  // TODO: map these steps to CUJs.
+
+  // Navigate to the initial url
+  await synthetics.executeStep('navigateToUrl', async function (timeoutInMillis = 30000) {
+    await page.goto(url, { waitUntil: ['load', 'networkidle0'], timeout: timeoutInMillis });
+  });
+
+  // Fill and search for an address.
+  await synthetics.executeStep('searchAddress', async function () {
+    await page.waitForSelector("[id='app']", { timeout: 30000 });
+    await page.screenshot();
+    const searchInput = await page.waitForSelector('input.mapboxgl-ctrl-geocoder--input');
```

Comment:
nit: Can you move some of these to constants?

Reply:
I'd rather leave it inline because it's only referenced once, here.